### ISSUE TITLE
Focus the iframes when moving through shot creation flow

### ISSUE THREAD

**Issue by 6a68** [mozilla-services/screenshots#4670]
Tab focus is lost when moving from preselection to selection iframe in the shot creation flow (from step 1 to step 2).

We should put the focus on one of the elements in the overlay when transitioning between frames.

**Comment by punamdahiya** [comment]
@6a68 looks like this one should be handled in web extension as well.

**Comment by chenba** [comment]
Not sure I quite understand the description.  Nothing is auto focused on the preselection frame, so I don't know what's lost or different when moving to the selection frame.  The one difference is that the buttons on the preselection frame have `tabindex`es whereas those on the selection frame do not.  Tabbing through the buttons work on both frames.
